docs/configuration.md

```diff
@@ -194,21 +180,40 @@ Example:
 
 The resulting package can be published to [PyPI].
 
-[PyPI]: https://pypi.org/
+[pypi]: https://pypi.org/
 [trove classifiers]: https://www.python.org/dev/peps/pep-0301/#distutils-trove-classification
 [pypa/trove-classifiers]: https://github.com/pypa/trove-classifiers
 
+##### Prerelease Versions
+
+The original `npm` package may feature a version number that includes a [SemVer 2.0][semver]-compliant prerelease
+identifer (e.g: `1.2.3-pre.4`). Python packages distributed to [PyPI] must however use a different format to express
+prerelease versions, as specified in [PEP-440]. In order to generate valid packages, only certain prerelease identifiers
+are accepted by `jsii-pacmak`, and are translated according to the following table:
+
+| Source Version (`npm`) | Python Version ([PEP-440]) | Notes                                  |
+| ---------------------- | -------------------------- | -------------------------------------- |
+| `X.Y.Z.dev.N`          | `X.Y.Z.devN`               | Development, iteration `N`.            |
+| `X.Y.Z.pre.N`          | `X.Y.Z.devN`               | Development, iteration `N`             |
+| `X.Y.Z.alpha.N`        | `X.Y.Z.aN`                 | Alpha release, iteration `N`           |
+| `X.Y.Z.beta.N`         | `X.Y.Z.bN`                 | Beta release, iteration `N`            |
+| `X.Y.Z.rc.N`           | `X.Y.Z.rcN`                | Release candidate, iteration `N`       |
+| `X.Y.Z.prN.M`          | `X.Y.Z.postN.devM`         | Post-release number `N`, iteration `M` |
```

Comment:
I don't see a use case for why we should support post releases or development versions of post releases. They should be new patch/minor versions, or new pre-release identifiers.

I can only find this in Python and even PEP-440 discourages them -

> The use of post-releases to publish maintenance releases containing actual bug fixes is strongly discouraged. In general, it is better to use a longer release number and increment the final component for each maintenance release.

> Creating post-releases of pre-releases is strongly discouraged, as it makes the version identifier difficult to parse for human readers. In general, it is substantially clearer to simply create a new pre-release by incrementing the numeric component.

Reply:
I mean I don't intend on those post-releases being published at all. I conventionally use `-pr<pr-id>.0` version suffix for PR-bound pre-releases (i.e: the artifacts that would be produced if that PR was to be merged).

So I found it actually mapped kind of neatly (this is the post-release of this thing for pr `<pr-id>`, sequence `0`).

Would it be "better" to you if I'd simply map `X.Y.Z-prN.0` to `X.Y.Z.devN` (and fail if the sequence is non-0)?

Reply:
> Would it be "better" to you if I'd simply map X.Y.Z-prN.0 to X.Y.Z.devN (and fail if the sequence is non-0)?

Better yes, but still doesn't work. 

Do we have a use case for PR-bound pre-releases? What's the value over using `dev` prefix with an auto-inc sequence instead?

Reply:
Alright - dropping support for this, will re-instate if actually needed.